relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -193,51 +201,99 @@ fn scrub_file(description: &str) -> Option<String> {
     }
 }
 
-/// Normalize the domain and port of a URL.
+/// Scrub a `Url::Host` object. For domain names, run the string through a scrubber. For IP addresses, return the IP address as-is.
+///
+/// # Arguments
+///
+/// * `host` - `Url::Host`
+///
+/// # Returns
+///
+/// A host `String`, or `None` if scrubbing fails.
+pub fn scrub_host(host: Host<&str>) -> Cow<'static, str> {
```

Comment:
```suggestion
pub fn scrub_host(host: Host<&str>) -> Cow<'_, str> {
```

No need to confine this to `'static`. You can now return the host name contained in `Host` without cloning.